functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -39,7 +43,10 @@ var profile *performancev2.PerformanceProfile
 func Test5LatencyTesting(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	//TODO Skip the suite before setup steps in case the executable is not found
+	if _, err := os.Stat(testExecutablePath); os.IsNotExist(err) {
+		testlog.Infof("The test executable file %q does not exist, skipping the suite.", testExecutablePath)
+		t.Skip()
```

Comment:
Is there a valid case where we expect the executable to be missing? 
If the reason for missing executable is due to something bad that happened, and not an expected scenario I think we should fail here and not skip.

Reply:
If we fail and do not skip it can be done in the ```BeforeSuite```

Reply:
the executable is generated via a make target, if something interrupts the creation of the binary then it would mostly be external from the tests, thus it tests should not fail as their intent is to run the provided binary with different environment variables. 
the make target to build the execfile: https://github.com/openshift-kni/performance-addon-operators/blob/release-4.10/Makefile#L115